README.md

```diff
@@ -30,29 +30,45 @@ inference.
     ```
     sudo apt-get install libturbojpeg
     sudo apt-get install python3-tk
-    virtualenv ./toolbox_env -p python3
-    source ./toolbox_env/bin/activate
-    pip install -r requirements.txt
     ```
 
-3. Build COCO API
+3. Create virtual environment
 
     ```
-    cd external/cocoapi
-    2to3 . -w
-    cd PythonAPI
-    python setup.py build_ext --inplace
+    bash tools/init_venv.sh
     ```
 
-4. Update PYTHONPATH environment variable
+4. Start to work
 
     ```
     cd /<path_to_working_dir>/training_toolbox_tensorflow
-    export PYTHONPATH=`pwd`:`pwd`/external/models/research/slim:`pwd`/external/models/research:`pwd`/external/cocoapi/PythonAPI:$PYTHONPATH
+    . venv/bin/activate
     ```
 
+## Tests
+In virtual environment run the Nose:
+
+```
+cd training_toolbox
+nosetests
+```
+
+or if you are going to use the OpenVino toolkit:
+
+```
+cd training_toolbox
+export OPEN_VINO_DIR=<PATH_TO_OPENVINO>
+nosetests
+```
+    
+Note: if you have install the OpenVino toolkit after creating a virtual environment then you have to recreate one to install required packages for the Model Optimizer into one.
```

Comment:
Could you please add the reference  to "Setup Training Toolbox for TensorFlow.3" on the text `recreate one`?

Reply:
Done